web/themes/custom/novel/templates/layout/header.html.twig

```diff
@@ -1,162 +1,150 @@
-          alt="Logo image of libary"
-        />
-      </div>
-    </a>
-  </div>
-  <div class="header-menu">
-    <nav class="header-menu--first">
-      <div>
-        <div class="header-menu--navigation--mobile">
-          <div
-            class="pagefold-parent-small header-menu--navigation--button header-button"
-            id="header-menu--open"
-          >
-            <div class="pagefold-triangle-small"></div>
-            <img src="/themes/custom/novel/assets/dpl-design-system/icons/basic/icon-menu.svg" alt="List of bookmarks" />
-          </div>
-          <div class="header-menu--navigation--logo">
-            <div class="logo-fallback-text">
-              <p class="logo-fallback-text--name">Lyngby-Taarbæk</p>
-              <p class="logo-fallback-text--libraries">Bibliotekerne</p>
+
+  <header class="header">
```

Comment:
Please look into indentation and whitespace in this file.

All the content seems to be indented and there is a blank line at the start of the file.

Reply:
👍